src/cr/cube/slices.py

```diff
@@ -676,7 +645,9 @@ def means(self):
     @lazyproperty
     def ndim(self):
         """int count of dimensions, unconditionally 0 for a Nub."""
-        return 0
+
+        # The ndim on the underlying _scalar is also unconditionally 0
+        return self._scalar.ndim
```

Comment:
same here. If we're not expecting variation, we should keep it a literal value here.